### ISSUE TITLE
Does not work with nouislider package

### ISSUE THREAD

**Issue by ondrique** [dshemendiuk/gulp-npm-dist#2]
Hi,

thanks for the great plugin. 

I am using it on a second project and encountered an issue today with usage with [nouislider](https://www.npmjs.com/package/nouislider).

Error I get is:

```
{ Error: ENOENT: no such file or directory, scandir 'D:\xampp7\htdocs\template_clothes\node_modules\nouislider\dist'
    at Error (native)
    at Object.fs.readdirSync (fs.js:953:18)
    at readLibFilesRecursively (D:\xampp7\htdocs\template_clothes\node_modules\gulp-npm-dist\index.js:63:12)
    at module.exports (D:\xampp7\htdocs\template_clothes\node_modules\gulp-npm-dist\index.js:75:5)

...
```

The problem IMO is as follows:

- package does not have a dist directory
- but it has a main in package.json ( "main": "distribute/nouislider") - note "distribute" folder name
- you look for the occurence of /dist in the mainFile path. In fact, it should be false but you get the position of /dist in a string containing /distribute

After changing line 49 to `distFolderPos = mainFile.lastIndexOf('/dist/'); `, it works as expected. (not sure if it does not mess other stuff up, though)

Cheers,

Ondrej

**Comment by dshemendiuk** [comment]
You are right.
I've added this fix to 0.1.6 version. 

Thank you, @ondrique!

**Comment by ondrique** [comment]
Thank you!
